lib/between_meals/knife.rb

```diff
@@ -73,6 +74,18 @@ def cookbook_upload_all
       exec!("#{@knife} cookbook upload -a -c #{@config}", @logger)
     end
 
+    def berks_cookbook_upload_all
+      @cookbook_dirs.each do |path|
+        Dir.chdir(path)  # http://stackoverflow.com/a/808285/830426
+        cookbook_dirs=Dir["*"].reject{|o| not File.directory?(o)}
```

Comment:
we'll be committing our rubocop configuration shortly so you can just rubocop this stuff, but, in the meantime, sorry, I have to lint you. :)

* Spaces around `=`
* Single quote around strings without interpolation
* Spaces around `{` and `}`
* Prefer `!` to `not`

In other words:

```
cookbook_dirs = Dir['*'].reject { |o| !File.directory?(o) }
```